Setup/UpgradeSchema.php

```diff
@@ -56,14 +56,20 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             ]
         );
 
+        $setup->getConnection()->dropColumn(
+            $setup->getTable('quote'),
```

Comment:
We will lose the backend order type for old orders on upgrade if we do it this way, right?
I think a safer approach is to write a migration function that'll run if upgrading from bolt version <=2.6.0.

Reply:
@matt-thomason this field is not propagated to the order table, it exists in quote table only and its sole purpose was defining the right redirect url, storefront vs admin. This functionality is completely preserved with the new field.

Reply:
That's a good point, thanks 👀